File: model/Money.java
package model;

import model.common.PocketObject;

public class Money extends PocketObject {
  private double amount;

  public Money(int id, double amount) {
    super(id);
    this.amount = amount;
  }

  public double getAmount() {
    return amount;
  }

  public void setAmount(double amount) {
    this.amount = amount;
  }

  @Override
  public String toString() {
    return "Money{" +
        "amount=" + amount +
        '}';
  }
}

File: model/Money.js
import PocketObject from './common/PocketObject.js';

class Money extends PocketObject {
    constructor(id, amount) {
        super(id);
        this._amount = amount;
    }

    getAmount() {
        return this._amount;
    }

    setAmount(amount) {
        this._amount = amount;
    }

    toString() {
        return `Money{amount=${this._amount}}`;
    }
}

export default Money;
